common/source/docs/common-msp-osd-overview.rst

```diff
@@ -0,0 +1,124 @@
+.. _common-msp-osd-overview:
+
+=======
+MSP OSD
+=======
+
+ .. image:: ../../../images/msp_dji_fpv_goggles.jpeg
+    :target: ../_images/msp_dji_fpv_goggles.jpeg
+
+ .. image:: ../../../images/msp_dji_goggles_re.jpeg
+    :target: ../_images/msp_dji_goggles_re.jpeg
+ 
+ArduPilot supports the special MSP OSD protocol which allows displaying flight data on the DJI goggles, much like with external MAVLink OSDs or the internal integrated OSD in many flight controllers.
+
+Features
+========
+ 
+ - ArduPilot currently supports many, but not all, of the OSD panel items as noted in the section below.
+ - Changing display units other than metric and imperial are not currently supported.
+ - Multiple screens and remote switching of those screens is not supported.
+ - Warning levels for RSSI, Voltage, etc. currently not supported
+
+Configuration
+=============
+
+To enable MSP OSD, set the following parameters (using SERIAL port 2 as the port to attach to the DJI Air unit using both TX and RX lines):
+
+ - :ref:`OSD_TYPE<OSD_TYPE>` = 3
+ - :ref:`SERIAL2_PROTOCOL<SERIAL2_PROTOCOL>` = 33
+ - :ref:`MSP_OPTIONS<MSP_OPTIONS>` = 0 (polling mode)
+
+
```

Comment:
We might add a note that external MSP osd has to be enabled on the goggles as well:
Under the DJI goggle’s SETTINGS, DISPLAY, CUSTOM OSD, you can turn the OSD ON or OFF.

Reply:
will do

Reply:
will do